### ISSUE TITLE
Integrate tokens from TrustWallet

### ISSUE THREAD

**Issue by danfinlay** [MetaMask/eth-contract-metadata#74]
Trust wallet has started their own token list, it is large and has images, we could periodically scrape it, or even move to it!

https://github.com/TrustWallet/tokens

**Comment by geopayme** [comment]
I agree it will avoid all the headaches that are created for token owners do to lack of coordination been Metamask and Trustwallet. Duplicate work can be avoided

**Comment by xrn** [comment]
2 things to notice:

1. here is new repo which should be used: https://github.com/trustwallet/assets/tree/master/blockchains/ethereum
2. They are using almost only png images

**Comment by 0xnikhil** [comment]
1. Is there any plan to have this going forward? Uniswap also uses this same repo for token logos. 
2. I know we use SVG for Metamask logos, would switching to trust wallet repo and using png have some trouble?
